README.md

```diff
@@ -2,10 +2,12 @@
 
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
 
-[![All Contributors](https://img.shields.io/badge/all_contributors-2-orange.svg?style=flat-square)](#contributors-)
+[all-contributors-logo]: https://img.shields.io/badge/all_contributors-2-orange.svg?style=flat-square "Number of contributors on All-Contributors"
```

Comment:
Unfortunately, the all contributors format seems to need this section unmodified, so I'm hesitant to add these changes. In fact, the all contributors bot wants to remove the extra lines in this section, but our linter (and the tests) seems to want to enforce an extra line in this section. For now i've "broken" the all contributors format in order for our tests to pass, but we may need to add an exception to the linter for the readme in order to not conflict with the all contributors bot again in the future

Reply:
Is there a way to validate if the AC bot is okay with the updates? If not or if it's too cumbersome then I'm all good to take this bit out. I was just trying to get the two badges to sit inline with each other.